### DISCUSSION TITLE
For the first time I'm seriously considering leaving GitHub

### DISCUSSION THREAD

**Discussion by Zeioth** [community/community#37456] [Other Feature Feedback, Questions, & Ideas]
As I stated the first time you tried to push changes to the start page, I'm here to work, every single day of my life. This is not a social media. This is not the place to make me swallow your 'sponsors'. Microsoft have plenty of other channels to do so, and I'm aware in the end GitHub is just a valuable source of machine learning data for you. So my advice is, don't screw it up. Don't make our life worse, let us work in peace. Revert this changes.

**Comment by ossanna16** [comment]
Hi there @Zeioth and welcome to our community! Thank you for your feedback 🙂
